js/my.js

```diff
@@ -232,3 +237,30 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  cmd = cmd.replace(/^\//, '');
+
+  var args = cmd.match(/(\S+) ?(\S+)? ?(.*)/);
+
+  if (args[1] === 'query' && args[2]) {
+    var to = args[2];
+    var msg = privMSG[to];
+
+    if (!msg) {
+      msg = privMSG[to] = new Tab({
+        chan: to,
+        client: client,
+        nick: nick,
+        host: host,
+      });
+
+      chans.push(to);
+    }
+
+    document.getElementById("container").showCard(chans.indexOf(to) + 1);
+
+    return;
```

Comment:
There's no need to have an empty return at the end of a function in JS.  JS functions that have empty return statements or no return statements always return `undefined`.  You only really see empty return statements if returning early from a function body.